guide/payments/introduction.md

```diff
@@ -0,0 +1,86 @@
+
+## Bitcoin is extensible, it's not just a payment network
+
+It should be noted, that bitcoin is not just a mechanism to make simple payments but it is a complete global financial infrasutructure for moving value without intermediaries. Each transaction is like a bit of code, which means it can be programmed — for example you can set various spending conditions like requiring multiple parties to agree prior funds being able to move. Its even possible to restrict funds from being moved until a certain time.
+
+## Bitcoin is secure and accessible to all
+
+Some people may have the perception that bitcoin is anonymous. This isn't quite accurate because all transactions are public so anyone can see the entire history of transfers that ever happened going back to the very first one on [2009-01-03](https://blockstream.info/tx/4a5e1e4baab89f3a32518a88c31bc87f618f76673e2cc77ab2127b7afdeda33b).
+
+#### Why are transactions public?
+
+They are public so that anyone can have the ability to verify the entire history of transactions themselves. We call this a permissionless system. No one needs to ask for permission to send a transaction or to build experiences on top of bitcoin.
+
+#### What about privacy?
+
+Even though all transactions are stored publicly, there is no personally identifiable information attached to them. Privacy is still super important though since every time you make a payment the person receiving can see where you funded that transaction from, as you would be able to trace which address they spend those funds.
+
+We should inform our users of these risks but also provide mechanisms that help them gain an acceptable amount of privacy[^1].
+
+<<<<<<< HEAD
+
+## Bitcoin is not too slow for payments
+
+Transactions made on-chain can take minutes or days to confirm — depending on the fee you pay. This is because they get secured by the entire network of nodes and miners which exchange transactions they receive from wallets and other services with each other. Every 10 minutes miners validate the payments within the transactions and publish a block. For this service, they typically take the transactions with the highest fees.
+
+This is may not be suitable if you want to make a small payment to a vendor at your local market — but if you're making or receiving a payment from someone accross the world the benefits are made very clear as the current banking and financial system can be quite restrictive, and expensive for some.
+
+### Lightning Payments — A micro-payment network for every day payments
+
+To meet the demands of our modern economy — movement of money needs to be borderless, secure, fast, and cheap.
+
+The Lightning Network is part of the bitcoin ecosystem that will power this economy. It allows for near-instant payments at much lower fees. This is possible because extends bitcoin in a way that does not record each payment on-chain.
+
+#### A Network of Payment Channels and Settlement 
+
```

Comment:
Probably some sort of visual of LN/Channels is needed here, as at this point it gets quite hard to follow up if you're newbie imo, especially since this is intro page.

Reply:
Agreed. Maybe describe it as another peer-to-peer network on top of Bitcoin. It's more flexible and allows for direct payments that don't require the whole network to confirm and process them. And that's how it manages to be faster and cheaper. Or something like that.

Reply:
@pavlenex yesss time for abacus beads? 😄 

@GBKS check the following paragraphs. I attempted to explain why it is faster there, and talk about two people locking up funds on-chain, then sending back and forth with one another. Then talk about final settlements on-chain.

Reply:
Quite honestly abacus beads were so helpful when I tried to understand how channels works so go for it :smile:

Reply:
Abacus time!

(added an illustration)